.ci/Jenkinsfile

```diff
@@ -188,7 +188,11 @@ def generateStep(version){
 }
 
 def releasePackages(){
-  withSecretVault(secret: 'secret/apm-team/ci/apm-agent-python-twine',
+  def vault_secret = (env.REPO_URL == 'https://upload.pypi.org/legacy/') ? 'secret/apm-team/ci/apm-agent-python-pypi-prod' : 'secret/apm-team/ci/apm-agent-python-pypi-test'
+  if (env.REPO_URL == 'https://upload.pypi.org/legacy/'){
```

Comment:
After @v1v's comments, lines L192:194 are not needed, as L191 uses the ternary operator to evaluate and assign the variable

Reply:
yeah, this whole "suggested changes" thing in github comments didn't really do what I think it would do. Thanks for catching it!